src/shared/ui/Components/CustomCheckBox/ui/CustomCheckBox.tsx

```diff
@@ -1,7 +1,7 @@
-import { TouchableOpacity } from 'react-native';
-import React, { useState } from 'react';
-import { styles } from './CustomCheckBox.style';
-import { CustomImage } from '../../CustomImage/ui/CustomImage';
+import { TouchableOpacity } from "react-native";
+import React, { useState } from "react";
+import { styles } from "./CustomCheckBox.style";
+import { CustomImage } from "../../CustomImage/ui/CustomImage";
 
 
```

Comment:
do we use it in project??? i'm not, so if you either just delete whole component